arm9/overlays/24/src/ov24_02254840.c

```diff
@@ -0,0 +1,254 @@
+#include "global.h"
+#include "bg_window.h"
+#include "unk_0200CA44.h"
+#include "gf_gfx_loader.h"
+#include "overlay_24.h"
+
+extern void Poketch_InitApp(void *func1, void *func2);
+extern u32 ov20_02254130(void *, u32, void *, void*, u32);
```

Comment:
these should also have param names, if you don't know you can use param0, param1, etc